tests/conftest.py

```diff
@@ -430,3 +462,157 @@ def plates_lookup_with_samples(samples, priority_samples):
 @pytest.fixture
 def plates_lookup_without_samples(samples, priority_samples):
     return PLATES_LOOKUP_WITHOUT_SAMPLES
+
+
+@pytest.fixture
+def mocked_rabbit_channel(app):
+    with app.app_context():
+        mocked_broker = MagicMock()
+        with patch("lighthouse.classes.services.warehouse.Broker", return_value=mocked_broker):
+            mocked_channel = MagicMock()
+            mocked_broker.__enter__.return_value = mocked_channel
+
+            yield mocked_channel
+
+
+@pytest.fixture
+def cherrytrack_mock_run_info(
+    app, mocked_responses, run_id, cherrytrack_run_info_response, cherrytrack_mock_run_info_status
+):
+    run_url = f"{app.config['CHERRYTRACK_URL']}/automation-system-runs/{run_id}"
+
+    mocked_responses.add(
+        responses.GET,
+        run_url,
+        json=cherrytrack_run_info_response,
+        status=cherrytrack_mock_run_info_status,
+    )
+    yield
+
+
+@pytest.fixture
+def cherrytrack_mock_source_plates_status():
+    return HTTPStatus.OK
+
+
+@pytest.fixture
+def cherrytrack_mock_run_info_status():
+    return HTTPStatus.OK
+
+
+@pytest.fixture
+def cherrytrack_mock_source_plates(
+    app,
+    mocked_responses,
+    source_barcode,
+    cherrytrack_source_plates_response,
+    cherrytrack_mock_source_plates_status,
+):
+    source_plates_url = f"{app.config['CHERRYTRACK_URL']}/source-plates/{source_barcode}"
+    mocked_responses.add(
+        responses.GET,
+        source_plates_url,
+        json=cherrytrack_source_plates_response,
+        status=cherrytrack_mock_source_plates_status,
+    )
+    yield
+
+
+@pytest.fixture
+def cherrytrack_run_info_response(run_id):
+    return {
+        "data": {
+            "id": run_id,
+            "user_id": "ab1",
+            "liquid_handler_serial_number": "aLiquidHandlerSerialNumber",
+        }
+    }
+
+
+@pytest.fixture
+def cherrytrack_source_plates_response(run_id, source_barcode):
+    return {
+        "data": [
```

Comment:
We shouldn't bloat this file with test data, it is already a large file.

Reply:
Done